Kitodo/src/main/webapp/pages/activeUsers.xhtml

```diff
@@ -1,45 +0,0 @@
-<!--
- *
- * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
- *
- * This file is part of the Kitodo project.
- *
- * It is licensed under GNU General Public License version 3 or later.
- *
- * For the full copyright and license information, please read the
- * GPL3-License.txt file that was distributed with this source code.
- *
--->
-
-<ui:composition
-        template="/WEB-INF/templates/baseListView.xhtml"
-        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
-        xmlns:h="http://xmlns.jcp.org/jsf/html"
-        xmlns:p="http://primefaces.org/ui">
-    <ui:define name="contentHeader">
-        <h3>#{msgs.activeUsers}</h3>
-    </ui:define>
-
-    <ui:define name="pageTabView">
-        <p:tabView id="activeUsersTabView">
-            <p:tab id="activeUsersTab" title="#{msgs.userList}">
-                <h:form id="activeUsersForm">
-                    <p:poll interval="10" update="tableActiveSessions"/>
-                    <p:dataTable id="tableActiveSessions" value="#{SessionForm.activeSessions}" var="userItem">
-                        <p:column id="userName" headerText="#{msgs.user}">
-                            <h:outputText value="#{userItem.userName}"/>
-                        </p:column>
-                        <p:column id="lastRequest" headerText="#{msgs.lastAccess}">
-                            <h:outputText value="#{userItem.lastRequest}"/>
-                        </p:column>
-                    </p:dataTable>
-                </h:form>
-            </p:tab>
-        </p:tabView>
-    </ui:define>
-
-    <ui:define name="breadcrumbs">
-        <li><h:link value="#{msgs.startPage}" outcome="start"/></li>
-        <li><a href="">#{msgs.activeUsers}</a></li>
```

Comment:
Probably this message is right now not used anymore an can be removed.

Reply:
The `activeUsers` message is still used in line 172 of `SearchForm.java`, so I didn't remove it.